pipeline/plugin_runners.go

```diff
@@ -769,21 +794,30 @@ func (kind foRunnerKind) String() string {
 // OutputRunner interfaces.
 type foRunner struct {
 	pRunnerBase
-	pluginType string
-	config     CommonFOConfig
-	matcher    *MatchRunner
-	ticker     <-chan time.Time
-	inChan     chan *PipelinePack
-	h          PluginHelper
-	retainPack *PipelinePack
-	leakCount  int
-	encoder    Encoder // output only
-	useFraming bool    // output only
-	canExit    bool
-	kind       foRunnerKind
-	pConfig    *PipelineConfig
-	lastErr    error
-}
+	pluginType          string
+	config              CommonFOConfig
+	matcher             *MatchRunner
+	ticker              <-chan time.Time
+	inChan              chan *PipelinePack
+	backChan            chan *PipelinePack
+	h                   PluginHelper
+	retainPack          *PipelinePack
+	leakCount           int
+	encoder             Encoder // output only
+	useFraming          bool    // output only
+	canExit             bool
+	useBuffering        bool
+	kind                foRunnerKind
+	pConfig             *PipelineConfig
+	lastErr             error
+	bufReader           *BufferReader
+	stopChan            chan bool
+	processMessageCount int64
+	dropMessageCount    int64
```

Comment:
Move these two items to the top of the struct to avoid alignment issues on 32 bit systems